test/dynamics/solvers/test_fixed_step_solvers.py

```diff
@@ -329,6 +329,69 @@ def solve(self, rhs, t_span, y0, max_dt, t_eval=None):
         return scipy_expm_solver(rhs, t_span, y0, max_dt, t_eval)
 
 
+class TestScipyExpmSolver_magnus2(TestFixedStepBase):
+    """Tests for scipy_expm_solver with magnus order 2."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Setup constants."""
+
+        cls.c1 = 0.5 - np.sqrt(3) / 6
+        cls.c2 = 0.5 + np.sqrt(3) / 6
+        cls.c3 = np.sqrt(3) / 12
+
+        super().setUpClass()
+
+    def take_step(self, rhs, t, y, h):
+        """In this case treat rhs like a generator."""
+        A1 = rhs(t + self.c1 * h)
```

Comment:
use snake case for variable names here and elsewhere (ie A1 -> a1)

Reply:
i'm don't see the changes right now. were they pushed?

Reply:
my bad - I thought this comment was in the non-test code, and I changed it there. I'll change it in the tests now too.